Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/PublicWebsiteSettings.java

```diff
@@ -0,0 +1,40 @@
+package edu.asu.spring.quadriga.web.publicwebsite;
```

Comment:
That's the wrong package. Only controllers serving the public webpages should go here. Just create a new subpackage in 'web' called 'settings' and put the class in there.